ingest/processors/corelight.go

```diff
@@ -191,7 +186,7 @@ func (c *Corelight) getTagTs(mp map[string]interface{}) (tag string, ts time.Tim
 func emitLine(ts time.Time, headers []string, mp map[string]interface{}) (line []byte, ok bool) {
 	bb := bytes.NewBuffer(nil)
 	var f64 float64
-	fmt.Fprintf(bb, "%.3f", float64(ts.UnixNano())/1000000000.0)
+	fmt.Fprintf(bb, "%.5f", float64(ts.UnixNano())/1000000000.0)
```

Comment:
Looking at actual zeek logs, they appear to use 6 digits of precision:

```
1663714154.384267	CXGv4RcQXWSFXRzGd	192.168.0.77
```

Reply:
fixed